### ISSUE TITLE
Allowing for private data/information to become less secure.

### ISSUE THREAD

**Issue by DJJones66** [mkwatson/chat_any_site#3]
Your 3rd motivation raised valid concerns with regard to private data, I feel you should continue this mindset through out the project then and use a local LLM and VectorDB this way you have zero risk of your data being out in the wild.

**Comment by mkwatson** [comment]
I agree having a local LLM could be a useful option. As it exists now, even using an external LLM and making a network request, none of your private data should be made public because the network isn’t training on it, only using it for inference.
